src/Accelerators/NNPA/Compiler/NNPACompilerUtils.cpp

```diff
@@ -149,6 +149,7 @@ void addPassesNNPA(mlir::OwningOpRef<mlir::ModuleOp> &module,
         normalizeMemRefsPasses(pm);
         // Optimizations at ZLow.
         pm.addPass(zlow::createZLowRewritePass());
+        pm.addPass(zlow::createZLowRewriteFinalPass());
```

Comment:
I think it would be better to add the new transformation patterns in the existing `createZLowRewritePass` pass.  Is it possible to do that ?

Reply:
New transformation patterns are added into the existing `createZLowRewritePass`. Thank you.